api/v1alpha1/constants.go

```diff
@@ -27,4 +27,6 @@ const (
 	ClusterAutoscalerVariableName = "clusterAutoscaler"
 	// ServiceLoadBalancerVariableName is the Service LoadBalancer config patch variable name.
 	ServiceLoadBalancerVariableName = "serviceLoadBalancer"
+	// CarenNamespace is the namespace that caren is deployed in.
+	CarenNamespace = "caren-system"
```

Comment:
Having a default here is a little tricky here since `caren-system` is the namespace when publishing the charts, but `default` is the default when installing via the Helm chart.
Would it be alot of work to be more explicit and fail if the env is not set? Or maybe sticking with `default` since thats closer to what Kubernetes behaves like, ie an empty namespace == `default`?

WDYT?

Reply:
actually we don't even use this any more so i'll just remove it

Reply:
:thinking: it should be there because of the downward API. i suppose we can use default as a back up here then.